server/implementation/src/main/java/io/smallrye/graphql/bootstrap/Bootstrap.java

```diff
@@ -389,18 +386,30 @@ private void addRootObject(GraphQLObjectType.Builder rootBuilder, Set<Operation>
         }
     }
 
-    private void addGroupedRootObject(GraphQLObjectType.Builder rootBuilder,
-            Map<Group, Set<Operation>> operationMap, String rootName) {
-        Set<Map.Entry<Group, Set<Operation>>> operationsSet = operationMap.entrySet();
+    private String prettyString(String value) {
+        return value.substring(0, 1).toUpperCase() + value.substring(1);
+    }
+
+    private void addNamespacedRootObject(GraphQLObjectType.Builder mutationBuilder,
```

Comment:
The parameters can also be for queries, can't they? Maybe better rename it to, e.g., `operationBuilder`/`namespaceOperations`/`operationType`?